solution.cpp

```diff
@@ -1,23 +1,121 @@
 /* ----------------------------------------------------------------------------
- * Copyright &copy; 2015 TODO(your name) <TODO(your email)>
- * Released under the [MIT License] (http://opensource.org/licenses/MIT)
- * ------------------------------------------------------------------------- */
+* Copyright &copy; 2015 Thomas Collins <thomas_collins@csu.fullerton.edu>
+* Released under the [MIT License] (http://opensource.org/licenses/MIT)
+* ------------------------------------------------------------------------- */
 
 /**
- * TODO: program description
- */
+* Program that converts user inputted integers into Roman Numerals.
+*/
 
-// TODO: `#include`s and function prototypes
+#include <iostream>
+using std::endl;
+using std::cin;
+using std::cout;
+
+#include <string>
+using std::string;
+
+string intToRoman(int i);
 
 // ----------------------------------------------------------------------------
 
-int main() {
-    // TODO
+int main()
+{
+	int i;
+	string s = "";
+	string repeat = "ayy lmao";
+
+	cout << "Welcome! This program converts integer numbers between 0 and\n4000 to Roman numerals." << endl;
+
+	while (repeat != "n" && repeat != "N")
+	{
+		
+		cout << "\nPlease enter the integer you would like to convert." << endl; // max of 4000
+		cin >> i;
 
-    return 0;  // success
+		if (i < 0 || i >= 4000)
+			cout << "\nInvalid input, please enter number between 0 and 4000." << endl;
+		else
+			cout << intToRoman(i) << endl;
+
+		cout << "\nWould you like to convert another integer to a Roman numeral? Enter anything to\nrepeat, or enter n to close." << endl;
+		cin >> repeat;
+
+	}
+
+	return 0;  // success
 }
 
 // ----------------------------------------------------------------------------
 
-// TODO: other functions
+string intToRoman(int i) // fat function that could likely be waayyy more streamlined.
```

Comment:
it could be more streamlined (as you've no doubt seen by now), but probably not without using at least slightly more advanced concepts.  overall, this method is probably my favorite.